Makefile

```diff
@@ -0,0 +1,43 @@
+EENVS  = PACKAGE_FILE="libgit.el"
+EENVS += PACKAGE_LISP="libgit.el"
+EENVS += PACKAGE_TESTS="$(wildcard test/*.el)"
+EENVS += PACKAGE_ARCHIVES="gnu"
+EMAKE  = $(EENVS) emacs -batch -l emake.el $(EMACS_OPTS) --eval "(emake (pop argv))"
+
+.PHONY: emacs clean test
+
+libgit2:
+	git submodule init
+	git submodule update
+
+build/libegit2.so: libgit2
+	mkdir build
+	cd build && cmake .. && make
+
+test: EMACS_OPTS=-L build/ --eval "(require 'libegit2)"
```

Comment:
Just `-l libegit2` should be sufficient.

Reply:
I'll make this change in a new PR pending vermiculus/emake.el#18